src/elements/Input/Input.js

```diff
@@ -88,7 +88,13 @@ function Input(props) {
   )
   const rest = getUnhandledProps(Input, props)
   const ElementType = getElementType(Input, props)
-  const inputProps = _.pick(props, htmlInputPropNames)
+  const inputProps = {
+    ..._.pick(props, htmlInputPropNames),
+    // React gives a warning if no onChange is given to a controlled input. We
+    // handle the onChange on the wrapping element, so we don't actually need
+    // onChange on the input itself but let's pass a noop to suppress the warning.
+    onChange: _.noop,
```

Comment:
_I think_ that is true?  Just make sure all the examples work/look how they should.  SUI relies partially on the classNames _and_ partially on the input props to do what it does for the Input component.

There are gonna be some exceptions for things like `disabled`, which adds opacity to the input.  It can't be both a class and an attribute or it is doubly transparent.  Also, `readOnly` I believe must be on the input to prevent entries, but not required as a className.